main.tf

```diff
@@ -28,8 +28,9 @@ locals {
 }
 
 resource "aws_s3_bucket" "bucket" {
-  bucket = var.bucket_name
-  acl    = "private"
+  bucket              = var.bucket_name
+  acl                 = "private"
+  object_lock_enabled = var.object_lock
```

Comment:
given we set this via `aws_s3_bucket_object_lock_configuration` does object lock also need to be defined here?

Reply:
yes you're right. I'll fix this

Reply:
I think Changelog would be nice on this repo